frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -213,9 +214,15 @@ object Interpreter {
     }
   }
 
+  private val projectBound = CommandsMessages.messages.filter {
+    case (name, CommandMessages(args, _)) => args.exists(_.name == "project")
+  }.map(_._1).mkString(" ")
```

Comment:
The filter-map combo can be replaced with `collect`:

```scala
private val projectBound = CommandsMessages.messages.filter {
  case (name, CommandMessages(args, _)) if args.exists(_.name == "projects") => name
}.mkString(" ")
```

Reply:
Thanks @Duhemm ! I knew there was to be something in the library to do this in one go